lastpass-cli/plan.sh

```diff
@@ -0,0 +1,50 @@
+pkg_name=lastpass-cli
+pkg_origin=core
+pkg_version=1.3.3
```

Comment:
FYI, when I build this, it reports its version as `1.3.3.GIT`

I don't think it's a problem, but I filed https://github.com/lastpass/lastpass-cli/issues/509 because it looks like a `version` file is missing from the release tarball.

Reply:
To be clear, I don't think anything needs changed in this PR. Just a FYI that I noticed this.

Reply:
Its probably up to the build system to generate this file. So we should add it as part of the `do_prepare()` or similar.

Reply:
I compared this with the brew installation on my machine, and it reports the same `1.3.3.GIT`, so this seems to be consistent with their other install methods.

Reply:
I was looking at [this line of script](https://github.com/lastpass/lastpass-cli/blob/master/LASTPASS-VERSION-GEN#L12) that suggests that a `version` file is supposed to be in the tarball (and it's not) - so I think it needs fixed in their tar release process.